zowe/zowe-cli
# Fix failing CLI system tests and related bugs

**What It Does**

This PR fixes CLI system tests that were failing non-intermittently.
Most fixes involved changes only to test files, but a few involved fixing bugs in the CLI and z/OS Files SDK.

**How to Test**
Verify that the following bugs are fixed:
- `--range` option ignored on `zowe files view uss-file` command
  - Run the command with `--range 0,1` to display only the first line of a USS file
- `--binary` option ignored by commands that upload and download USS directories when ".zosattributes" file is used
  - Upload the directory `packages/cli/__tests__/zosfiles/__system__/upload/dtu/__data__/command_upload_dtu_zosattributes_dir` with the flags `--recursive --binary` and verify that files not defined in ".zosattributes" file like "picCopy.png" are uploaded as binary
- `--include-hidden` option ignored by `zowe files upload dir-to-uss` without the `--recursive` option
  - Upload the directory `packages\cli\__tests__\zosfiles\__system__\upload\dtu\__data__\command_upload_dtu_dir\dir_with_hidden_files` with the flag `--include-hidden` and verify that "dot files" are uploaded

**Review Checklist**
I certify that I have:


**Additional Comments**
